extensions.json

```
@@ -971,6 +976,11 @@
       "id": "Nightrains.robloxlsp",
       "repository": "https://github.com/NightrainsRbx/RobloxLsp"
     },
+    {
+      "id": "njpwerner.autodocstring",
+      "respository": "https://github.com/NilsJPWerner/autoDocstring",
+      "version": "0.5.4"
```

No need to pin a specific version if we publish straight from the default branch.

```suggestion
      "respository": "https://github.com/NilsJPWerner/autoDocstring"
```